pkg/monitor/cluster/clusteroperatorconditions_test.go

```diff
@@ -62,7 +62,11 @@ func TestEmitClusterOperatorConditions(t *testing.T) {
 				},
 			},
 		},
-	})
+	},
+		&configv1.ClusterVersion{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "version",
+			}})
```

Comment:
Becaus formatting it is not immediately clear where this object belongs. Maybe something like this?

```go
configcli := fake.NewSimpleClientset(
	&configv1.ClusterOperator{
		// ...
	},
	&configv1.ClusterVersion{
		ObjectMeta: metav1.ObjectMeta{
			Name: "version",
		},
	},
)
```